Platform/AlderlakeBoardPkg/Library/FspsUpdUpdateLib/FspsUpdUpdateLib.c

```diff
@@ -319,6 +319,7 @@ GetMaxCpuPciePortNum (
   }
 }
 
+#if FeaturePcdGet (PcdTccEnabled)
```

Comment:
This condition is already checked when the function is being called? Why do you have to build out completely?

Reply:
It was added to fix build errors caused due to Tcc FSP UPDs which are used in function definition.